### PR TITLE
feat(locale): added ES locale

### PR DESCRIPTION
**Pull request by carlos55ml**
feat(locale): added ES locale to game menu

### CODE REVIEW

File: locale/es.json [hunk 1]
```diff
@@ -5,11 +5,11 @@
     },
     "restarter": {
         "start_timeout": "Se ha agotado el tiempo de espera para el arranque.",
-        "crash_detected": "Caída del servidor detectada.",
```
Comment (BombayV) [inline locale/es.json:8]: Why did you remove the accent from caida and the dot at the end?

File: locale/es.json [hunk 2]
```diff
@@ -5,11 +5,11 @@
...
+        "crash_detected": "Caida del servidor detectada",
         "hang_detected": "Suspensión detectada",
         "partial_hang_warn": "Debido a una suspensión parcial, el servidor se reiniciará en 1 minutos. Por favor, desconectate ahora.",
         "partial_hang_warn_discord": "Debido a una suspensión parcial, **%{servername}** se reiniciará en un minuto.",
-        "schedule_reason": "Reinicio programado a las %{time}",
```
Comment (BombayV) [inline locale/es.json:12]: Isn't this supposed to be lowercase?

File: locale/es.json (hunk 1 above)
Comment (carlos55ml) [reply locale/es.json:8]: Maybe. typo and bad review. thx for fixing

File: locale/es.json (hunk 2 above)
Comment (carlos55ml) [reply locale/es.json:12]: no its not. thank you for fixing.

**Comment by tabarra** [comment]
Thank you very much for your PR and sorry for the delay!  
On the last update (v4.8.0) we finally released custom locale for the menu, meaning it is finally ready to be translated.  
Unfortunately, we changed a lot in the file structure so the best course of action is to close this PR.  
  
If you are still interested in translating the menu, please make a new PR and follow the new guide:  
https://github.com/tabarra/txAdmin/blob/master/docs/translation.md#contributing
